src/lib/isPostalCode.js

```diff
@@ -57,7 +58,7 @@ const patterns = {
 
 export const locales = Object.keys(patterns);
 
-export default function (str, locale) {
+export default function(str, locale) {
```

Comment:
Travis failing because of this, you need to unstage this particular change; we're using [standardJS](https://standardjs.com/) formatting guidelines.